contracts/FeeManager/FeeManager.sol

```diff
@@ -73,4 +72,14 @@ contract FeeManager is IFeeManager, Ownable {
     ) private view returns (IPartnerConfiguration) {
         return _partnerManager.getPartnerConfiguration(partner);
     }
+
+    function _getPartnerOwnerAccount(
+        address partner
+    ) private view returns (address) {
+        return _partnerManager.getPartnerOwnerAccount(partner);
+    }
+
+    function getBalance(address partner) external view returns (uint256) {
```

Comment:
aShould we consider not making this method public if we're going to make the `balances` not public either way?
Not sure there is much difference between a public balances field and a public method for getting the balance with an address parameter